examples/desktop/crud/get_sharedkey.c

```diff
@@ -82,7 +82,7 @@ int main() {
     atclient_atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_DEBUG, "Created shared key\n");
   }
 
-  if ((ret = atclient_atkey_to_string(atkey, atkeystr.str, atkeystr.len, &atkeystr.olen)) != 0) {
+  if ((ret = atclient_atkey_to_string(&atkey, atkeystr.str, atkeystr.len, &atkeystr.olen)) != 0) {
```

Comment:
propogated a change to the api, atclient_atkey_to_string takes a pointer to an atkey rather than the memory of the atkey itself - the other previously mentioned files with formatting also have this change